grammar.js

```diff
@@ -368,12 +368,10 @@ module.exports = grammar({
       seq(
         "case",
         field("subjects", $.case_subjects),
-        "{",
-        field("clauses", $.case_clauses),
-        "}"
+        field("clauses", $.case_clauses)
```

Comment:
As-is this change will make the `{`/`}` belong to the `clauses` field. Instead I think we should just make the case clauses optional:

```suggestion
        "{",
        optional(field("clauses", $.case_clauses)),
        "}"
```

Reply:
I might be doing this wrong, but making that change and also updating `case_clauses` to `case_clauses: ($) => repeat($.case_clause),` produces this warning:  ```The rule `case_clauses` matches the empty string.
```.

That's why originally included the braces in that part. I think I saw that in the `ecma` grammar. Open to handling this in a different way if possible though!

Reply:
If the `clauses` field is optional then there's no need to change the `repeat1` to `repeat` in `case_clauses`